client/reader/stream/x-post.jsx

```diff
@@ -205,7 +205,7 @@ class CrossPost extends PureComponent {
 							</a>
 						</h1>
 					) }
-					<Emojify>{ this.getDescription( post.author.first_name ) }</Emojify>
+					<Emojify>{ post.author && this.getDescription( post.author.first_name ) }</Emojify>
```

Comment:
```suggestion
					{ post.author && <Emojify>{ this.getDescription( post.author.first_name ) }</Emojify> }
```

maybe? otherwise we'll try to Emojify nothing.

Reply:
Sure!